src/App.js

```diff
@@ -30,27 +30,33 @@ class App extends Component {
     };
 
     // Uncomment this to spawn a single test item
-    // const testItem = this.spawnItem(Date.now());
-    // this.state.items.push(testItem);
+    const testItem = this.spawnItem(Date.now());
+    this.state.items.push(testItem);
 
     // Uncomment this to automatically spawn new items
     this.enableSpawner();
 
     console.log(this.state);
   }
 
-  onItemClicked = () => {
+  onItemClicked = (type) => {
     // Fill this in for Wave 3!
+    if (type === 'litter'){
+    this.setState({ points: this.state.points +1 });
   }
+}
 
   render() {
     const items = this.state.items.map((item, i) => {
       return <GameItem
                height={item.height}     // Height - used for a CSS style to position on the screen
                layer={100 + i}          // Layer - used for a CSS style to show items on-top of bg
                key={item.id}            // Key - to help React with performance
+               type={item.type}
+               callBack ={this.onItemClicked}
```

Comment:
In a larger project, you may have more than one callback function. Giving this a clearer name will help in those situations.